lib/cache/cache_test.go

```diff
@@ -840,6 +840,7 @@ func (s *CacheSuite) TestTokens(c *check.C) {
 
 // TestClusterConfig tests cluster configuration
 func (s *CacheSuite) TestClusterConfig(c *check.C) {
+	ctx := context.Background()
```

Comment:
Since you're adding a `ctx` variable here, could you replace all the `context.TODO` occurrences in this function with `ctx` for the sake of consistency?